iocore/net/SSLClientUtils.cc

```diff
@@ -66,6 +66,7 @@ verify_callback(int preverify_ok, X509_STORE_CTX *ctx)
   netvc = SSLNetVCAccess(ssl);
 
   if (netvc != nullptr) {
+    netvc->callHooks(TS_EVENT_SSL_SERVER_VERIFY_HOOK);
```

Comment:
This should prob. be moved up in the function -- you won't reach it if openssl validation fails or if this cert is part of the chain (second part might be desirable since it stops the callback from triggering multiple times).

If it's in the beginning, someone can implement plugin side validation logic and override the 
ATS core side checks